tests/integration/pure_cairo_vm/cheatcodes/deploy/deploy_cairo_cheatcode_test.py

```diff
@@ -0,0 +1,37 @@
+from pathlib import Path
```

Comment:
- [ ] Please add tests for `prepare` cheatcode
- [ ] Please add tests for `warp` cheatcode
- [ ] Please add tests for `roll` cheatcode
- [ ] Please add a test with two interleaving declare->prepare->deploy flows